Components/FlightControl/FlightTask.cpp

```diff
@@ -64,13 +64,13 @@ void FlightTask::Run(void * pvParams)
 
         // Since FlightTask is so critical to managing the system, it may make sense to make this a Async task that handles commands as they come in, and have these display commands be routed over to the DisplayTask
         // or maybe HID (Human Interface Device) task that handles both updating buzzer frequencies and LED states.
-        GPIO::LED1::On();
-        GPIO::LED2::On();
-        GPIO::LED3::On();
+//        GPIO::LED1::On();
+//        GPIO::LED2::On();
+//        GPIO::LED3::On();
         osDelay(500);
```

Comment:
Let's make sur  at least one of these is blinking every second so we know the board is running right (doesn't necessarily need to be in HDI, can be in WatchdogTask)